### ISSUE TITLE
build global script missing from npm package

### ISSUE THREAD

**Issue by SteveALee** [paulrosen/abcjs#465]
`/dist` only contains a npmignore file not the actual built global script js files.

On your suggestion I decided to use npm to install and manage all my client dependencies and then copy the global scripts to my `_site` folder for inclusion in my index,html

However, abcjs seems to be the only one that does not actually supply the required scripts :)

The script are in the zip release archive however.

**Comment by SteveALee** [comment]
I tried bulding and seeing why the files are missing but cannot find anything; `bin` is populated by the build scripts. I guess the publish to npm process is replacing the files? Couldn't see where ther is though. Certainly not an npm script.

I also noted the source on 'master' is building beta-10 and could find to suitable branch or tag for the latest.

**Comment by paulrosen** [comment]
When using from the node ecosystem, just include it with:
```
import ABCJS from "abcjs";
```
Then you will have the same access to it that you did when you used the minimized version.

**Comment by SteveALee** [comment]
Except I'm not using a bundler (yet). I made decision to leave it untill the benfits make it sensible.

I did explore using browser native ES6 modules but there's a dependency order problems to iron out as modules defered - plus it's more work it work with npm packaged deps like abc.

I will be trying [snowpack](https://www.snowpack.dev/) soon which can use a [bundler for production](https://www.snowpack.dev/#bundle-for-production) builds, native ES6 for dev. I'll sort out module structure then.

Anyway, if your, enot plannign to provide the global scripts, that's finer. I'll continue to use a copy in my source tree and include the npm package so my IDe tells me a new version is available.
